### ISSUE TITLE
Dired keybindings fail b/c of already bound ";" key used as prefix

### ISSUE THREAD

**Issue by meliache** [emacs-evil/evil-collection#318]
When I call `(evil-collection-dired-setup)`, I get the following error:
```
Debugger entered--Lisp error: (error "Key sequence ; d starts with non-prefix key ;")
```
I fixed it for me by commenting out the following bindings, which I don't use much anyway:
```
    ;; encryption and decryption (epa-dired)
    ";d" 'epa-dired-do-decrypt
    ";v" 'epa-dired-do-verify
    ";s" 'epa-dired-do-sign
    ";e" 'epa-dired-do-encrypt
```
Seems that semicolon is already bound by evil to `evil-repeat-find-char`. But both bindings of evil-collection-dired and evil to this key seem to be years old when I looked at the git blame, so I don't know what suddenly caused the error.

**Comment by Ambrevar** [comment]
From `C-h k ;`

> ; runs the command evil-repeat-find-char (found in
> evil-motion-state-map),

So `;` is not bound in dired which is in evil-normal-state by default.
I suspect something is wrong with your config.  Can you reproduce with a
vanilla configuration?  If not, can you provide a recipe with a minimal
config?

**Comment by meliache** [comment]
Could not reproduce it in the vanilla configurations, seems actually something weird was going on in my configuration and looks like I bound `;` in dired, just had to run `C-h k ;` in dired-mode and I had it bound to `dired-subtree-remove`. I had this keybinding set in dired when I was using emacs without evil and wanted it to remain in evil for some time, so I had defined (in a kind of reduntant way)
```
(use-package dired
  :config
  (use-package dired-subtree
    :demand t
    :bind (:map dired-mode-map
                ("i" . dired-subtree-insert)
                (";" . dired-subtree-remove))
    :config
    (evil-define-key 'normal dired-mode-map
      "i" #'dired-subtree-insert
      ";" #'dired-subtree-remove)))
```
Recently I started using the `TAB` key provided by evil-collection for dired-subtree-toggle anyway, so I just removed these lines and everything worked for me. So for the PR without figuring that out first, for some reason it seemed to me first that the issue appeared after a package-update and thus couldn't be my fault, but as I mentioned above that can't be and probably was due to configuration changes on my side, also I was tired. Thanks to all for the awesome work btw.
